docs/src/features/features.md

```diff
@@ -33,7 +33,7 @@ The following tables show the status of Ghaf Platform features:
 | `aarch64` reference image | &#x2705; | `imx8qm`  | Based on NXP BSP, implemented as [nixos-hardware module](https://github.com/NixOS/nixos-hardware/tree/master/nxp)|
 | `x86` generic image | &#x2705; | `x86` | Generic x86 computer, based on generic [NixOS](https://nixos.org/). NOTE: requires device specific configuration.|
 | Native build      | &#x2705;         | `aarch64, x86`   | Remote `aarc64` nixos builders recommended |
-| Cross-compilation | &#x1f6A7; | `aarch64`  | Depends on NixOS `nixpkgs 23.05` support for cross-compilation |
+| Cross-compilation | &#x1f6A7; | `aarch64, riscv64`  | Depends on NixOS `nixpkgs 23.05` / `nixpkgs 22.11`(for riscv64) support for cross-compilation |
```

Comment:
Depending on `nixpkgs 22.11` in ghaf tree is problematic. What are the exact issues for `riscv64` support on `nixpkgs 23.05`? I've understood it has better cross-compilation support in general and the cross-compilation support will be improved also for ghaf dependencies on `aarch64`

Reply:
There are few issues I found in cross compilation for RiscV platform.
Listed the same in code also.
[BUG:235179](https://github.com/NixOS/nixpkgs/issues/235179)
[BUG:235526](https://github.com/NixOS/nixpkgs/issues/235526)

Reply:
We still have to address the ghaf `23.05`-only support issue.